### ISSUE TITLE
Docker Desktop integration is enabled with the default distro and it must but turned off.

### ISSUE THREAD

**Issue by MrR341-GER** [docker/for-win#13726]
### Description

I don't have docker desktop installed and want to install it with docker inside wsl.. but with the second command I get following error:

```powershell
PS C:\Windows\system32>
Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072;

PS C:\Windows\system32>
iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/ddev/ddev/master/scripts/install_ddev_wsl2_docker_inside.ps1'))

Docker Desktop integration is enabled with the default distro and it must but turned off.
In Zeile:32 Zeichen:5
+     throw "Docker Desktop integration is enabled with the default dis ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Docker Desktop ...but turned off.:String) [], RuntimeException
    + FullyQualifiedErrorId : Docker Desktop integration is enabled with the default distro and it must but turned o
   ff.

```
### Reproduce

In Powershell:
Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072;
iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/ddev/ddev/master/scripts/install_ddev_wsl2_docker_inside.ps1'))

### docker version

```bash
none
```

### docker info

```bash
none
```

### Diagnostics ID

none

**Comment by thaJeztah** [comment]
If you have docker desktop installed, you probably want to run the other variant of the script that's meant to be used in combination with Docker Desktop; https://github.com/ddev/ddev/blob/master/scripts/install_ddev_wsl2_docker_desktop.ps1

**Comment by MrR341-GER** [comment]
No, I don't have Docker Desktop installed and I want to install it **without** the docker desktop app.

**Comment by pasqual-artif** [comment]
Hi.. different Account - same person here:
Actually I just saw this in the script:
https://raw.githubusercontent.com/ddev/ddev/master/scripts/install_ddev_wsl2_docker_inside.ps1
```powershell
if (wsl bash -c "test -d /mnt/wsl/docker-desktop >/dev/null 2>&1" ) {
    throw "Docker Desktop integration is enabled with the default distro and it must but turned off."
}
```
But the folder doesn't exist if Docker Desktop isn't installed, so this doesn't make sense (to me).

And second:
If the ps-script is running as administrator, wsl -l -v is empty, because I'm the administrator and not the user.

To be more precise:

My user on my Windows machine is not an administrator and therefore running the script as administrator, won't list my normal users wsl distributions.
On the other hand, running ths script with normal privileges, results in permission-errors for obvious reasons.

Long story short:
I've not been able to install ddev with docker inside for a non administrative windows user
(I think I make a new issue for that last problem..)
